extensions/kube-object-event-status/renderer.tsx

```diff
@@ -19,45 +19,45 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { LensRendererExtension, K8sApi } from "@k8slens/extensions";
+import { Renderer } from "@k8slens/extensions";
 import { resolveStatus, resolveStatusForCronJobs, resolveStatusForPods } from "./src/resolver";
 
-export default class EventResourceStatusRendererExtension extends LensRendererExtension {
+export default class EventResourceStatusRendererExtension extends Renderer.LensExtension {
   kubeObjectStatusTexts = [
     {
       kind: "Pod",
       apiVersions: ["v1"],
-      resolve: (pod: K8sApi.Pod) => resolveStatusForPods(pod)
+      resolve: (pod: Renderer.K8sApi.Pod) => resolveStatusForPods(pod)
```

Comment:
We could make these less verbose with:

```ts
const K8sApi = Rendered.K8sApi;
```

Reply:
Or maybe better to split main/renderer via different module names to avoid verbosity? Wdyt?

```ts
import { K8sApi } from "@k8slens/extensions/main"
import { Components } from "@k8slens/extensions/renderer"
```

Reply:
I tried to do what Roman was suggesting, but was not able to figure it out.

Reply:
@jakolehm unfortunately because `K8sApi` is a `namespace` there is no destructuring currently, or even aliasing. See https://github.com/microsoft/TypeScript/issues/13135